src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs

```diff
@@ -142,6 +142,19 @@ public static bool IsWindows()
             return OSID.AnyWindows.MatchesCurrent();
         }
 
+        public static bool IslinuxWithIIS()
```

Comment:
Trying to understand what this is trying to determine?  Is this trying to determine whether this is Linux or not?  Or is trying to determine something else?  Is there a better name?

Reply:
I have update suitable name to 'IsNotLinuxWithIIS'.